### PR TITLE
Fix TYPO in README

### PR DESCRIPTION
**Pull request by HeavyWombat**
Fix TYPO in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,11 @@ A collection of Go based sample code.
 
 ## Overview
 
-This repository host minimal assets develop for the Golang runtime. These samples are used in [Shipwrigh/Build](https://github.com/shipwright-io/build) for testing purposes.
+This repository host minimal assets develop for the Golang runtime. These samples are used in [Shipwright/build](https://github.com/shipwright-io/build) for testing purposes.
```
Comment (qu1queee) [inline README.md]: can u keep the consistency of upper case on "build" , as in the later parts of this README, we write it on that way.

Comment (qu1queee) [reply README.md]: or viceversa, if you want "Build" to be "build", not strong opinions, it should just be consistent across the README.

**Review by qu1queee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HeavyWombat) [reply README.md]: Ha! Introducing a TYPO while trying to solve a TYPO. Classic IT.

**Review by qu1queee — APPROVED** [review]
lgtm

**Pull request merged**